zowe/zowe-cli
# Show strings instead of regexes for allowed values in CLI help

**What It Does**

Fixes #1863

**How to Test**

1. Run `zowe zos-files list data-set --protocol invalid` and verify that there are no `^` or `$` characters in the command response:
    ```
    You specified:
    invalid
    
    The value must match one of the following regular expressions:
    [ 'http', 'https' ].
    ```
2. Run `zowe auth logout apiml --help` and verify that there is no `^` character before "apimlAuthenticationToken":
    ```
       --token-type | --tt (string)
    
          The type of token to get and use for the API. Omit this option to use the
          default token type, which is provided by 'zowe auth login'.
          
          Allowed values: apimlAuthenticationToken.*, jwtToken, LtpaToken2
    ```

**Review Checklist**
I certify that I have:


**Additional Comments**

**Note:** Removing the `^` in front of "apimlAuthenticationToken" is purely cosmetic and should not change the behavior, because it will be added by `SyntaxValidator`:
https://github.com/zowe/zowe-cli/blob/59a4b1ea5fbdba0e7bf133daca25665077fa77fa/packages/imperative/src/cmd/src/syntax/SyntaxValidator.ts#L300-L304
